### PR TITLE
Create nytimes3xbfgragh.onion.xml

### LINKED ISSUE #13366: Make new ruleset for new New York Times Onion Service https://www.nytimes3xbfgragh.onion/
[//]: # (Thank you for reporting an issue to HTTPS Everywhere.)
[//]: # (We welcome input from users on improving this project.)
[//]: #
[//]: # (Please help us by following this issue template.)
[//]: # (You can delete all blank lines and all lines starting)
[//]: # (with the comment marker, such as this one.)
[//]: #
[//]: # (Delete all but one of the following "Type" lines.)
[//]: # (Leave in the line that best describes the issue you are reporting.)

Type: new ruleset

[//]: # (If you are reporting a ruleset/website problem, include the top-level)
[//]: # (domain below. For example, if you want to report an issue about)
[//]: # ("one.example.com" and "two.example.com", then the line below should)
[//]: # (be:)
[//]: #
[//]: # (Domain: example.com)
[//]: #
[//]: # (Be sure to remove the parenthesis and comment marker.  If you are only)
[//]: # (reporting an issue about "one.example.com", then the line below should)
[//]: # (be:)
[//]: #
[//]: # (Domain: one.example.com) 
[//]: #
[//]: # (Only include one top-level domain. If you have more than one top-level)
[//]: # (domain to report, such as both "example.com" and "example.org", open a)
[//]: # (new issue for each top-level domain.)
[//]: #
[//]: # (If you are not reporting a ruleset/website problem, you can delete the)
[//]: # ("Domain" line or leave it blank.)

Domain: nytimes3xbfgragh.onion

[//]: # (Include any other relevant information below. Thank you again for)
[//]: # (helping to improve HTTPS Everywhere.)

-------

URL: https://www.nytimes3xbfgragh.onion/

You can check that it's legit from the EV certificate that is in use.

### PR DESCRIPTION
**Pull request by Bisaloo**
fix #13366

### CODE REVIEW

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
There are other urls that pop up in the developer > network such as https://static01.graylady3jvrrxbe.onion/

**Comment by Bisaloo** [comment]
I think I will add it in another ruleset. This one is complex enough for an onion service IMO and it still looks under construction (so many requests to clearnet...)

File: src/chrome/content/rules/graylady3jvrrxbe.onion.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!--
+	For more NY Times ruleset, see NYTimes.xml
+
+-->
+<ruleset name="graylady3jvrrxbe.onion">
+
+	<target host="a1.graylady3jvrrxbe.onion" />
+		<test url="http://a1.graylady3jvrrxbe.onion/fonts/family/cheltenham/cheltenham-normal-400.ttf" />
+	<target host="typeface.graylady3jvrrxbe.onion" />
+		<test url="http://typeface.graylady3jvrrxbe.onion/fonts/nyt-cheltenham-extra-condensed-bold-700-normal.woff" />
```
Comment (I-have-50-open-PRs-to-HTTPS-EVERYWHERE) [inline src/chrome/content/rules/graylady3jvrrxbe.onion.xml]: Please add


```
<target host="static01.graylady3jvrrxbe.onion" />
<test url="http://static01.graylady3jvrrxbe.onion/images/2017/10/28/business/28ECON1/27ECON1-largeHorizontalJumbo.jpg" />
```

File: src/chrome/content/rules/nytimes3xbfgragh.onion.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	<target host="*.blogs.nytimes3xbfgragh.onion" />
+		<test url="http://lens.blogs.nytimes3xbfgragh.onion/" />
+		<test url="http://afterdeadline.blogs.nytimes3xbfgragh.onion/" />
+		<test url="http://open.blogs.nytimes3xbfgragh.onion/" />
+	<target host="cn.nytimes3xbfgragh.onion" />
+	<target host="et.nytimes3xbfgragh.onion" />
+	<target host="global.nytimes3xbfgragh.onion" />
+	<target host="graphics.nytimes3xbfgragh.onion" />
+		<test url="http://graphics.nytimes3xbfgragh.onion/subscriptions/components/img/icons/icons_gift.png" />
+	<target host="mobile.nytimes3xbfgragh.onion" />
```
Comment (I-have-50-open-PRs-to-HTTPS-EVERYWHERE) [inline src/chrome/content/rules/nytimes3xbfgragh.onion.xml]:
```
<target host="samizdat-graphql.nytimes3xbfgragh.onion" />
<test url="http://samizdat-graphql.nytimes3xbfgragh.onion/graphql/v2" />
```

Comment (Bisaloo) [reply src/chrome/content/rules/nytimes3xbfgragh.onion.xml]: Still different content http/https

File: src/chrome/content/rules/graylady3jvrrxbe.onion.xml (hunk 1 above)
Comment (Bisaloo) [reply src/chrome/content/rules/graylady3jvrrxbe.onion.xml]: Thanks! Added.

File: src/chrome/content/rules/nytimes3xbfgragh.onion.xml (hunk 2 above)
Comment (I-have-50-open-PRs-to-HTTPS-EVERYWHERE) [reply src/chrome/content/rules/nytimes3xbfgragh.onion.xml]: But nytimes3xbfgragh.onion makes use of the https:// version, search its source code for `samizdat`

File: src/chrome/content/rules/graylady3jvrrxbe.onion.xml (hunk 1 above)
Comment (Bisaloo) [reply src/chrome/content/rules/graylady3jvrrxbe.onion.xml]: `^` and `www` redirect to `http`.

Comment (I-have-50-open-PRs-to-HTTPS-EVERYWHERE) [reply src/chrome/content/rules/graylady3jvrrxbe.onion.xml]: @Bisaloo Didn't notice the redirect, thanks for the remark!

**Comment by Giltyhub** [comment]
@Bisaloo this has been up for quiet some time, will you revisit it or can I take it over?

**Comment by Bisaloo** [comment]
@Giltyhub, feel free to take over.

**Comment by ivysrono** [comment]
@Bisaloo https://travis-ci.org/EFForg/https-everywhere/jobs/585897922#L346

**Comment by ivysrono** [comment]
https://travis-ci.org/EFForg/https-everywhere/jobs/586071662#L353

**Comment by Bisaloo** [comment]
Seems to work locally for me. Should I just whitelist this ruleset?

**Comment by ivysrono** [comment]
IMO, split them to another ruleset.

**Pull request merged**